### ISSUE TITLE
H2O API documentation

### ISSUE THREAD

**Issue by proyb6** [h2o/h2o#1654]
I understand the rapid changes to H2O code, is there a place we can refer to the API since reading from code can be hard for someone who come across new web server

**Comment by emanwebdev** [comment]
Yes, You mean adding CDoc comments into the code so it-s possible to automatically generate up to date documentation

**Comment by GuacheSuede** [comment]
Yes, that would be great, CDOC !
